src/main/scala/za/co/absa/hyperdrive/trigger/scheduler/executors/spark/SparkExecutor.scala

```diff
@@ -50,6 +53,13 @@ object SparkExecutor {
       }) match {
         case Seq(first) =>
           updateJob(jobInstance.copy(applicationId = Some(first.id), jobStatus = getStatus(first.finalStatus)))
+        case _
+            // It relays on the same value set for sparkYarnSink.submitTimeout in multi instance deployment
+            if jobInstance.jobStatus == JobStatuses.Submitting && jobInstance.updated
+              .map(lastUpdated => ChronoUnit.MILLIS.between(lastUpdated, LocalDateTime.now()))
+              .exists(_ < sparkConfig.yarn.submitTimeout + 60000) =>
```

Comment:
Ok, but `60000` should be at least a class variable :D or even better an application property

Reply:
Done